### PR TITLE
Check that all opeartor images are from Redhat

### PR DESCRIPTION
**Pull request by petrpinkas**

### CODE REVIEW

**Review by ssulka — COMMENTED** [review]
I didnt find any issues and code looks just fine. Submitting just general feedback so Kristian could also do review. But I think its good to go.

File: test/support/acceptance.go [hunk 1]
```diff
@@ -27,22 +28,24 @@ func ParseSnapshotImages() (SnapshotMap, error) {
 	return snapshotImages, nil
 }
 
-func ParseOperatorImages(helpContent string) OperatorMap {
+func ParseOperatorImages(helpContent string) (OperatorMap, OperatorMap) {
 	const minimumValidMatches = 3
 	re := regexp.MustCompile(`-([\w-]+image)\s+string[^"]+default "([^"]+)"`)
 	matches := re.FindAllStringSubmatch(helpContent, -1)
-	images := make(OperatorMap)
+	operatorSnapshotImages := make(OperatorMap)
```
Comment (kdacosta0) [inline test/support/acceptance.go:35]: Suggestion: Rename `operatorSnapshotImages` to `operatorTasImages` for consistency with the rest of the code.

File: test/support/acceptance.go [hunk 2]
```diff
@@ -27,22 +28,24 @@ func ParseSnapshotImages() (SnapshotMap, error) {
...
 		if len(match) >= minimumValidMatches {
 			key := match[1]
 			value := match[2]
-			if key == "client-server-image" || key == "trillian-netcat-image" { // not interested in these
+			if slices.Contains(OtherOperatorImageKeys(), key) {
+				operatorOtherImages[key] = value
 				continue
 			}
-			images[key] = value
+			operatorSnapshotImages[key] = value
```
Comment (kdacosta0) [inline test/support/acceptance.go:45]: Suggestion: Rename `operatorSnapshotImages` to `operatorTasImages` for consistency with the rest of the code.

File: test/support/acceptance.go [hunk 3]
```diff
@@ -27,22 +28,24 @@ func ParseSnapshotImages() (SnapshotMap, error) {
...
 		}
 	}
-	return images
+	return operatorSnapshotImages, operatorOtherImages
```
Comment (kdacosta0) [inline test/support/acceptance.go:48]: Suggestion: Rename `operatorSnapshotImages` to `operatorTasImages` for consistency with the rest of the code.

**Review by kdacosta0 — APPROVED** [review]
/lgtm

**Pull request merged**